### ISSUE TITLE
Entities attached to camera not parsed correctly (potential regression in 1.6.0)

### ISSUE THREAD

**Issue by floe** [aframevr/aframe#5537]
**Description:**

I've just upgraded my code from 1.4.2 to 1.6.0 and noticed that my camera-relative "avatar head" isn't properly parsed anymore. The code snippet is below; when I use this code (which worked nicely on 1.4.2) with 1.6.0, then I get the following console warnings:

```
core:component:warn Unknown property `width` for component `geometry`. [common.js:113:9]
core:component:warn Unknown property `height` for component `geometry`. [common.js:113:9]
core:component:warn Unknown property `depth` for component `geometry`. [common.js:113:9]
core:component:warn Unknown property `color` for component `material`. [common.js:113:9]
core:component:warn Unknown property `height` for component `geometry`. [common.js:113:9]
```

(The avatar head is supposed to be only visible in external cameras, obviously, but those seem to be broken as well. In any case, if I put the `<a-entity avatar ...` block outside of the camera, then it is rendered correctly.)

- A-Frame Version: 1.6.0
- Platform / Device: Firefox 126
- Reproducible Code Snippet or URL:
```xml
<a-entity camera="active: true" look-controls wasd-controls position="0 1.6 0">
   <a-entity avatar position="0 0 0.3" rotation="0 180 0" scale="2 2 2">
      <a-entity geometry="primitive: box; width: 0.2; height: 0.2; depth: 0.2" material="color: blue" position="0 0 0"></a-entity>
      <a-entity geometry="primitive: cylinder; height:0.001; radius: 0.02" material="color: white" position=" 0.05 0.05 0.1" rotation="90 0 0"></a-entity>
      <a-entity geometry="primitive: cylinder; height:0.001; radius: 0.02" material="color: white" position="-0.05 0.05 0.1" rotation="90 0 0"></a-entity>
      <a-entity geometry="primitive: cylinder; height:0.1; radius: 0.05" material="color: white" position="0 -0.03 0.05" rotation="60 0 0" id="mouth"></a-entity>
   </a-entity>
</a-entity>
```

**Comment by dmarcos** [comment]
Can you create a simple full runnable example with https://glitch.com/~aframe ? Thanks

**Comment by vincentfretin** [comment]
For the warnings, it should be fixed on master with #5526

**Comment by floe** [comment]
Thanks @vincentfretin, just realized that the warnings sent me on a wild goose chase, the geometry is actually rendered correctly. 🤦 I had to fix my offscreen canvas context setup, but that's completely unrelated. Sorry for the noise...
